File: javascript/Q10.js
var arr=[1,2,3,4,5];
var oddcount=0;
var evencount=0;

function odd(){
    for (i=0;i<arr.length;i++){
        if(arr[i]%2==0){
            evencount++
        }else{
            oddcount++
        }
    }
    console.log("Even:",evencount,"Odd:",oddcount)
}

odd()

File: C - language/Q10.c
#include <stdio.h>

int main() {
    int array[5]={1,2,3,4,5};
    int even;
    int odd;
    int len=sizeof(array)/sizeof(array[0]);
    int i;
    for (i=0;i<len;i++){
        if (array[i]%2==0){
            even=even+1;
        }else {
            odd=odd+1;
        }
    }
    printf("Even:%d  Odd:%d",even,odd);
    return 0;
}